scss/_buttons.scss

```diff
@@ -119,19 +119,19 @@
       @include button-variant(
         $value,
         $value,
-        $hover-background: shade-color($value, $btn-hover-bg-shade-amount),
-        $hover-border: shade-color($value, $btn-hover-border-shade-amount),
-        $active-background: shade-color($value, $btn-active-bg-shade-amount),
-        $active-border: shade-color($value, $btn-active-border-shade-amount)
+        $hover-background: shift-color($value, $btn-hover-bg-shade-amount),
```

Comment:
These changes weren't asked but seems more consistent to me. Might bring some breaking changes.